sdk/turing/router/config/router_version.py

```diff
@@ -53,3 +56,22 @@ def get_config(self) -> RouterConfig:
         :return: a new RouterConfig instance containing attributes of this router version
         """
         return RouterConfig(**self.to_dict())
+
+    @classmethod
+    def create(cls, config: RouterConfig, router_id: int) -> RouterVersion:
+        """
+        Creates a new router version for the router with the given router_id WITHOUT deploying it
+
+        :param config: configuration of router
+        :param router_id: router id of the router for which this router version will be created
+        :return: the new router version
+        """
+        version = turing.active_session.create_router_version(
+            router_id=router_id,
+            router_version_config=config.to_open_api(use_generic_config=True)
```

Comment:
We can construct the `RouterVersionConfig` here manually for now (in line with the comment above to remove `use_generic_config`).

Reply:
Okay I'll leave a comment there to say that it'll be removed once the refactoring is complete.

Reply:
I've simplified this further by just extracting the `RouterVersionConfig` directly from `RouterConfig`, so there's now no need for this temporary thing.